src/shared/ui/OrderSummary/OrderSummary.module.scss

```diff
@@ -0,0 +1,82 @@
+@use '../../styles/helpers/variables.scss' as *;
+@use '../../styles/helpers/mixins.scss' as mixin;
+@use '../../styles/helpers/functions.scss' as func;
+
+.summary {
+  width: func.toRem(489);
+
+  @media (max-width: $m-viewport) {
+    width: func.toRem(300);
+  }
+
+  @media (max-width: $s-viewport) {
+    width: func.toRem(250);
+  }
+
+  @media (max-width: $xs-viewport) {
+    width: 100%;
+    border-top: func.toRem(1) solid #eaebed;
+    position: fixed;
+    bottom: 0;
+    background-color: #ffff;
+  }
+}
+
+.summaryTitle {
+  font-weight: 600;
+  font-size: func.toRem(24);
+  border-bottom: func.toRem(1) solid $border;
+  padding-bottom: func.toRem(15);
+  margin-top: func.toRem(20);
+
+  @media (max-width: $xs-viewport) {
+    padding: func.toRem(24) func.toRem(16);
+    margin: 0;
+  }
+}
+
+.summaryContent {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  font-size: func.toRem(16);
+
+  @media (max-width: $xs-viewport) {
+    padding: func.toRem(16) func.toRem(16) func.toRem(24) func.toRem(16);
+    // padding-bottom: func.toRem(24);
+  }
+}
+
+.contentTitle {
+  font-weight: 700;
+  margin: 0;
```

Comment:
use span instead of p so u dont need margin here it should reset in base close to p,h1,h2,h3... {margin:0}